Allow to send notifications to flagship apps (#3686)
cozy/cozy-stack: docs/flagship.md

- serving the `index.html` of a webapp
- starting the OAuth danse of a konnector (`/accounts/:slug/start`)
- starting the reconnect BI flow (`/accounts/:slug/reconnect`).

## Notifications

When a notification is sent via the push channel, the stack will first try to

> sent it to the dedicated mobile app. If there are no dedicated apps with a push

Reviewer: ```suggestion
send it to the dedicated mobile app. If there are no dedicated apps with a push
```

> token, the stack will then try to sent the notification to the flagship apps.

Reviewer: ```suggestion
token, the stack will then try to send the notification to the flagship apps.
```

> If it fails again, the stack then fallback on email.

Reviewer: ```suggestion
If it fails again, the stack then fallbacks to email.
```

If the notification is sent to the flagship app, the title is modified to
preprend the application name. The name is taken from the `appName` field of
the additional parameters (`data`). It allows the user to have more context

> when reading the notification on its mobile.

Reviewer: ```suggestion
when reading the notification on their mobile.
```
